app/src/main/java/a/a/a/Dp.java

```diff
@@ -169,7 +168,24 @@ public void a(iI iIVar, String str) {
      * @throws Exception Thrown if dexing had problems
      */
     public final void a(String outputPath, ArrayList<String> dexes) throws Exception {
-        dexLibraries(outputPath, dexes);
+        try {
+            if (!Integer.parseInt(this.a) >= 21) {
+                dexLibraries(outputPath, dexes);
+            } else {
+                Iterator it = dexes.iterator();
+                while (it.hasNext()) {
+                    String dexPath = it.next();
```

Comment:
![image](https://user-images.githubusercontent.com/74397286/142868002-302ad78a-8d26-4fb8-8813-ff0bd905733a.png)
how to fix this sir

Reply:
use the generic type `Iterator<Spring>`